nfcreader/build.gradle

```diff
@@ -8,10 +21,16 @@ android {
         minSdkVersion 19
         targetSdkVersion 29
         versionCode 1
-        versionName "1.0"
+        versionName getVersionName()
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         consumerProguardFiles 'consumer-rules.pro'
+        Properties versionProps = new Properties()
+        versionProps.load(new FileInputStream(file('config.properties')))
+        def properties_flurryKey = versionProps['FLURRY_API_KEY']
```

Comment:
As the property file is `config.properties` the variable name should be configProps not versionProps
and for `properties_flurryKey ` the standard naming convention for Groovy variables is camelCase as you can see the names in the remaining of the file, so please rename it

Reply:
sure noted .Done

Reply:
You forget to change `versionProps ` 😅